assets/script.js

```diff
@@ -210,14 +195,19 @@ function quitGame() {
 
 // Controls the timer
 function startTimer() {
-    var timeInterval = setInterval(function(){
+    timeInterval = setInterval(function(){
```

Comment:
making `timeInterval` global is what lets us clear the interval in the new `stopTimer` function.